core/pl/geohash.pm.sdoc

```diff
@@ -104,4 +104,15 @@ if (1 << 32) {
 *ghe = \&geohash_encode;
 *ghd = \&geohash_decode;
 
+*gh_box = sub {
```

Comment:
What's the reason for defining the functions like this? I just followed convention here--is it related to being in a BEGIN block?